app/src/main/java/cs465/illinois/edu/dogonthequad/CreateMeetupLocation.java

```diff
@@ -0,0 +1,23 @@
+package cs465.illinois.edu.dogonthequad;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+
+import com.google.gson.Gson;
+
+public class CreateMeetupLocation extends Activity {
```

Comment:
I'd recommend ending activity class names with `Activity`, which seems to be the Android standard.